opentelemetry-api/src/trace/span_context.rs

```diff
@@ -120,9 +120,9 @@ impl TraceId {
     }
 }
 
-impl From<[u8; 16]> for TraceId {
```

Comment:
Might make sense to keep the old `From` impl and introduce the new ones as an addition?

Reply:
I don't think it makes a lot of sense to keep the old `From<[u8; 16]>` for at least three reasons:
1. there is an unspecified ambiguity about the byte-ordering of those bytes
2. there are already explicit public methods (`from_bytes` and `to_bytes`) for handling these conversions
3. primitive types like `u64` also have `from_xx_bytes` method, but do not implement `From<[u8; 8]>` presumably due to 1 above

Regarding 2 above, I'd think we should call those methods `from_be_bytes` and `to_be_bytes` to be consistent with std and give the caller a better hint that byte-order is big endian.

Reply:
Fair enough!